### ISSUE TITLE
Update examples to use the same Rust version used for the parser

### ISSUE THREAD

**Issue by Tartasprint** [pest-parser/book#36]
In pest-parser/pest#715 the Rust edition was updated to the 2021 edition.
I imagine it could be a good thing to the same with the examples given in the book.

This is the first time I contribute to a project, I am not sure how to proceed. I forked this repository and made the changes to adress this issue on [this branch](/Tartasprint/pest-book/tree/2021-edition-migration). Now I think I should do a  pull request, but since I'm not sure if those changes are wanted I prefered creating an issue. Please tell me if this the expected behaviour :D

**Comment by tomtau** [comment]
@Tartasprint yes, feel free to go ahead to open a PR. Maybe pest versions could be bumped in those examples as well?

**Comment by Tartasprint** [comment]
I can do that :)
Shouldn't I also update the `rust-version` field to be `1.59.0` according to [this pull request](https://github.com/pest-parser/pest/pull/850) ?

**Comment by tomtau** [comment]
@Tartasprint no, as I mentioned, that was for `cargo-msrv` (https://foresterre.github.io/cargo-msrv/), not pest itself
